k8s/production/cdash/sealed-secrets.yaml

```diff
@@ -0,0 +1,20 @@
+    db-port: AgCjQxnJwzl0bkRsBp+p8ygZ6SHXVrB8Fh0O2HRCgCk4OeFIvwuedjHMcJTnx6nKohDzqbUG0AUkc4sIpVZK3YLj/R9i4j/q9+j2b85enEH/FEau4I0RazD4U5usi3DSjiiVJvQ04/BSKFVD/Ia4CMuBPViuQnsRRHuG7h7nHJAG87qvAaQLlbnmlFtqNcc4Eqb2k1Ua+HRV7NqM6tzv2uyASPcTqLzoIbJ7PeI6NvybkzT2iWM8gmFPCt/SMCvXz4ljRFbhsJwJxoaKC/cZpF4CCUAgSU6Zz9406oQCuKFEhYyniWSMND/b/Rms8srDt2+8IFdahsyfNRW65zhu/82lioIHBoPZfMtX8GHTEYbF+Rs8CNWOplbCeyPRBvDonflC0aRVOPaWAnXGDNQ8IiP3Mot2RBbz6Exp79uQorA9QO9FkfxHEcdcwhPArepsL5gxciyacR2+LZd4xF3oczhqKArYWHMfGO3wxra7hGkF2f6SIUw99mhMVZp+YnTVxiTajIQxJP6P/AOwZuUdVK98IM+XSTIGrkT3TVxVNA/vPp5NTytBvzw0LZLn+2CBu190YpM8MAklanOvJRQEdbYqFBfG/PDmm9PAvhhqd4C/5Jqap6UssT3Wn8ecrjXN++eBWeAvQL2yWyRS2gLhjjl3BQTETaClan9gbJS1dNSztb+tFfqwB4bW/Y+s/2C5lKMHA0mv
+    db-type: AgBtX6FCT3TJe7AxXe7cZ68gMCqQVfzwNv+0fiCf28o0bgd8lT9O6ySLoG05xQRmf3q3LD5pExxY9dxcUrunjoUk1DpC0Ttrb78/7SwU8tdxVuwZUzwhmEJd11MF/qM2KxF9lO0OOjwpfwdJaVe6cVDmoNKGUG/m5GtpkuMHIkDta7cHnd05jsfvDnoNJdIDjHIrIqm7XJ+aeH5VoUEX4P31C40yqEnF/BTiu3mFnEg1UN8OYBuwq/yhrGmhGFSp3EygNwqbr759foe3ME6pAaznT8SFKFfoPHcj2VnbYNiTchnhkot6NIAfjfxKXvLa1qV57evYJLsULBkF7VEk8njmoGc7ljkRseHcmuKuRk3TkdwsIwaH11OeAvyYKrr/t7vE3gRc4kzxDrFkAqvrKp+/Kkwlfurzb6d9lBRElCrpb2SKTqpTWGTbBRCR5lXUVp2gw3gIDNcwvGdPkc/DtKWwWhEc5V7dYv9W4JNmrfaGUmlX4RrKBpouSH30D6hhzjznHW3KQSe3F6sMv9w4HYemcTM4Kzri57v/NIrn6sAaPwMKTZqGxBQCXex1OaOYKjW3ycI/Xfs2GfrjJEmwfO1A5R4qwnKTpNh+wZOYfJDE9WeQgJs3w9LA1RKVXoJfbOhdmYSTUV6sK9t/Fpg0DLgDtnjqjUVgWWZy8jBvD0yLG92RHn9QrS89jqo2+sL7WPGZiDay3w==
+    root-password: AgDTy47AEtzteGZqRl+OivJUJ/BFLHxZYLVaJ6H/LwPhEyGWhJVZz1VhEnSE3IFkK9avwE3VUvgteyzfBfe8/8X0sHIv3M6ZAQRqr9WyYqnQla64Olco5gvJptJRSgkt3CE+WRWxq/qMoyzuxacYaJsUyOIsq6V8dCMhGTzQEheJQUGfWhXIzxpkOMqcn72qmpAo1ZXWdAYrZfwOymvk0kOOXt/1DAdbPXg4euENgCJIuJahvUUkgOGzhJqYKyaBzKWbj6LBfL8Cw2hdw+ByljwFUegujoUZeeRjArgV6t4jd05z7O9nm5KMhv2cPhW1Dk6AcKkcmETcByYOWf/k6N42hGvxSybYCrsfEHQSTbphkWlEMjNk3Dw1/RJw1h2zK3tPsEMKJNudXvncZ374wa7CLLSqB0mWf+2dn1oM5Kc7cIUzUev4Ea1OfBxSJbdLtTDAkgMjMGL6O5Fyov2cGQ2Y0y8SOluhIkOvcMAjIIlEG59UX+KzhFD/+ob2J0LPxfeY+tgHe3se277r1qjCNAPOK4yMacgfR32jjrtOw2i55iEGCGQzbAmhdVHq+mhM6LFpv0qKaFLEiFEXn0vuiun0p0XMdfSAWrmzJCDlbwNO1Y9eKqhOXfWilvqiOBLmRpOr3sKsOYdWa+CFhnzI5OziEnB6Bje6nNmoJKuhwtSrpuZhsOzI2SF4yD32xjFybfwurdENNOF0fNVDSYXjbPmS
+  template:
+    metadata:
+      annotations:
+        kustomize.toolkit.fluxcd.io/reconcile: disabled
```

Comment:
Why disable flux? Don't we want changes to secrets to be synchronized with the github repo?

Reply:
A couple of things:

1. This is under `spec.template`, which means it'll be applied to the created/updated `Secret` resource, not the `SealedSecret` resource. So long as the `SealedSecret` resource is tracked, the secrets will get updated
2. Given the point above, I still wasn't sure if we should keep these annotations on the (non-sealed) secrets. All of the secrets currently have this annotation, so was just trying to keep things consistent. I don't know if this annotation even has any effect if the resource isn't checked into source, maybe you can provide some insight there.